eel/src/storage_persister.rs

```diff
@@ -416,33 +413,32 @@ impl<ChannelSigner: WriteableEcdsaChannelSigner> Persist<ChannelSigner> for Stor
         // Launch background task that handles persisting monitor remotely
         let data = data.encode();
         let storage = Arc::clone(&self.storage);
-        let _chain_monitor = match { self.chain_monitor.read().unwrap() }.upgrade() {
+        let chain_monitor = match { self.chain_monitor.read().unwrap() }.upgrade() {
             None => return ChannelMonitorUpdateStatus::PermanentFailure,
             Some(c) => c,
         };
+        let encrypted_data = match encrypt(&data, &self.encryption_key) {
+            Ok(d) => d,
+            Err(_) => {
+                error!("Failed to encrypt ChannelMonitor data");
+                return ChannelMonitorUpdateStatus::PermanentFailure;
+            }
+        };
 
-        // The idea is to deal with remote persistence using the following code, but a deadlock bug
-        // needs to be fixed first
-        /*self.runtime_handle.spawn(persist_monitor_remotely(
-            storage,
-            chain_monitor,
-            data,
-            channel_id,
-            update_id,
-        ));
-
-        ChannelMonitorUpdateStatus::InProgress*/
-        let retries = 20;
-        let encryption_key = self.encryption_key;
-        match std::thread::spawn(move || {
-            sync_persist_monitor_remotely(storage, &encryption_key, data, channel_id, retries)
-        })
-        .join()
-        {
-            Ok(Ok(_)) => ChannelMonitorUpdateStatus::Completed,
-            Ok(Err(_)) => ChannelMonitorUpdateStatus::PermanentFailure,
-            Err(_) => ChannelMonitorUpdateStatus::PermanentFailure,
-        }
+        // Launch thread instead of tokio task due to the use of blocking reqwest in our
```

Comment:
the use of blocking reqwest in our [???]

Reply:
It continues on the next line :)

Reply:
Oh, I'm sorry, this was poorly formatted in my review screen